internal/controller/authenticator.go

```diff
@@ -142,7 +147,7 @@ func IsRunningInCluster() bool {
 	return os.Getenv("KUBERNETES_SERVICE_HOST") != ""
 }
 
-// SetUpOAuthConnection establishes a gRPC connection using OAuth credentials and logging the process.
+// Update SetUpOAuthConnection to use the helper function
```

Comment:
What does this comment mean?
Why replace a comment that follows the Golang style guide with one that doesn't?